Python/ceval.c

```diff
@@ -2472,6 +2476,27 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
             goto error;
         }
 
+        TARGET(RETURN_VALUE_QUICK) {
+            /* TODO(lpereira): Find a way to not have the code for this opcode to be
+             * a duplicate of the code for RETURN_VALUE. */
```

Comment:
I think that's a common pattern, not sure we care that much (if it's really exactly the same, the compiler can merge the two blocks :-). Then again, there is an [issue](https://bugs.python.org/issue45116) in the tracker that suggests the MS compiler refuses to inline this function because it's too large...) On the third hand, I don't understand why we need this if it's *exactly* the same?

Reply:
I've sent a new version that doesn't copy the code around, although it introduces a bit of redundant writes.  These aren't the same: the `QUICK` version doesn't expand the `TRACE_FUNCTION_EXIT()` and `DTRACE_FUNCTION_EXIT()` macros, skipping the tracing checks; it's the counterpart of what `START_FUNCTION` does, but on the function exit point.